src/channels/components/CannotDefineChannelsAvailabilityCard/CannotDefineChannelsAvailabilityCard.tsx

```diff
@@ -0,0 +1,28 @@
+import { Card, CardContent } from "@material-ui/core";
+import React from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+
+import CardTitle from "../../../components/CardTitle";
+
+const messages = defineMessages({
+  title: {
+    defaultMessage: "Availability",
+    description: "CannotDefineChannelsAvailabilityCard title"
+  },
+  subtitle: {
+    defaultMessage:
+      "You will be able to define availability of product after creating variants.",
+    description: "CannotDefineChannelsAvailabilityCard subtitle"
+  }
+});
+
+const CannotDefineChannelsAvailabilityCard: React.FC<{}> = () => (
```

Comment:
You can skip templating 👉

```suggestion
const CannotDefineChannelsAvailabilityCard: React.FC = () => (
```